athena-google-bigquery/src/main/java/com/amazonaws/athena/connectors/google/bigquery/BigQueryRecordHandler.java

```diff
@@ -127,7 +127,7 @@ private void handleStandardQuery(BlockSpiller spiller,
                                      List<QueryParameterValue> parameterValues,
                                      BigQuery bigQueryClient) throws Exception
     {
-        String projectName = configOptions.get(BigQueryConstants.GCP_PROJECT_ID);
```

Comment:
have we tested if BigQuery treats those project IDs sensitively?  meaning are they case sensitive?

Reply:
Hi @AbdulR3hman, yes, I have tested and please find attached testing doc for your reference. We can't create project with uppercase letters as per official documentation. Please refer attached document for more details.
[GBQ Project Id casing issue testing.docx](https://github.com/user-attachments/files/16140020/GBQ.Project.Id.casing.issue.testing.docx)